src/components/grid/index.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Box } from '@chakra-ui/react';
+
+import Grid from './index.tsx';
+
+<Meta
+  title="Layout/Grid"
+  component={Grid}
+  parameters={{
+    controls: {
+      sort: 'alpha',
+    },
+  }}
+  args={{
+    columns: 3,
+    spacing: 'md',
+  }}
+  argTypes={{
+    spacing: {
+      options: ['xs', 'sm', 'md', 'lg', 'xl', '2xl', '3xl', '4xl'],
+      control: { type: 'select' },
+    },
+    spacingY: {
+      options: ['xs', 'sm', 'md', 'lg', 'xl', '2xl', '3xl', '4xl'],
+      control: { type: 'select' },
+    },
+    spacingX: {
+      options: ['xs', 'sm', 'md', 'lg', 'xl', '2xl', '3xl', '4xl'],
+      control: { type: 'select' },
+    },
+    maxW: {
+      table: {
+        disable: true,
+      },
+    },
+  }}
+/>
+
+export const Template = ({ maxW, ...args }) => {
+  return (
+    <Box w="100%" maxW={maxW}>
+      <Grid {...args}>
+        <Box h="50px" bg="blue.100">
```

Comment:
Is it possible to use our theme values?  `4xl` -> 48px

```suggestion
        <Box h="4xl" bg="blue.100">
```

the same for the rest

Reply:
I'd rather have full control in the storybook and not rely on the theme. It's consistent with how we demo the stack